rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -0,0 +1,157 @@
+"""Images workflow functions that actually do the work."""
+
```

Comment:
Could you please add the similar header information as provided in https://github.com/rcsb/py-rcsb_exdb/blob/master/rcsb/exdb/wf/ExDbWorkflow.py#L1-L16 ?

(Obviously tailor it for you, and the Update section would just have one line saying something like, "Created class")